src/kiutils/symbol.py

```diff
@@ -426,7 +426,7 @@ class SymbolLib():
     Documentation:
         https://dev-docs.kicad.org/en/file-formats/sexpr-symbol-lib/
     """
-    version: Optional[str] = None
+    version: str = "20211014"
```

Comment:
Please change to 
```
version: str = KIUTILS_CREATE_NEW_VERSION_STR;
```

and add on top of the file:
```
from kiutils.misc.config import KIUTILS_CREATE_NEW_VERSION_STR
```

Reply:
Ah, brilliant.  Thank you for the note.